aws-predictions/src/main/java/com/amplifyframework/predictions/aws/service/AWSRekognitionService.java

```diff
@@ -226,6 +246,60 @@ void recognizeCelebrities(
         return celebrities;
     }
 
+    private IdentifyTextResult detectPlainText(Image image) throws PredictionsException {
+        DetectTextRequest request = new DetectTextRequest()
+                .withImage(image);
+
+        // Read text in the given image via Amazon Rekognition
+        final DetectTextResult result;
+        try {
+            result = rekognition.detectText(request);
+        } catch (AmazonClientException serviceException) {
+            throw new PredictionsException(
+                    "Amazon Rekognition encountered an error while detecting text.",
+                    serviceException, "See attached service exception for more details."
+            );
+        }
+
+        StringBuilder fullTextBuilder = new StringBuilder();
+        List<String> rawLineText = new ArrayList<>();
+        List<IdentifiedText> words = new ArrayList<>();
+        List<IdentifiedText> lines = new ArrayList<>();
+        for (TextDetection detection : result.getTextDetections()) {
+            String detectedText = detection.getDetectedText();
+            Geometry geometry = detection.getGeometry();
```

Comment:
Minimal savings from caching these. I think you're running out of width on lines 271 and 272 because `IdentifyEntitiesResultTransformers` is so long. Maybe we can rename that to something shorter?

Also you're starting at +12 from class indentation, method indentation, and `for` indentation.

Maybe the body of the `for` loop can be externalized to a helper? Among other things, that'd set you back to +8 indentation at least.

(Point in short, by the time you actually get to call `fromPoints`, it's pretty far to the right side of the screen.)

Reply:
> Maybe the body of the `for` loop can be externalized to a helper? Among other things, that'd set you back to +8 indentation at least.

It would still be +12 no?
```
for () {
    foo();
    bar();
    baz();
}
``` 
has the same indentation as
```
for () {
    doFooBarBaz();
}
```
(although definitely cleaner)

Meanwhile, I also moved the `IdentifiedText` builder into utility class for simplicity inside service implementation